pkg/daemon/ceph/osd/device.go

```diff
@@ -70,13 +71,22 @@ type Device struct {
 	Dir    bool   `json:"bool"`
 }
 
+// DesiredDevice keeps track of the desired settings for a device
+type DesiredDevice struct {
+	Name          string
+	OSDsPerDevice int
```

Comment:
Is there a good reason to want more than one OSD per device? It has been my understanding (assumption?) that would have no practical benefit. Unless this is referring to the number of osd-journal partitions on a device?

Reply:
when using NVMe devices it is actually recommended to create multiple OSDs. They can handle such high IO, multiple OSDs is the only way to saturate them. One OSD on NVMe becomes bound by the processor speed.

Reply:
TIL